harness/harness.rb

```diff
@@ -16,6 +16,14 @@
 
 system('mkdir', '-p', File.dirname(OUT_CSV_PATH))
 
+def yjit_enabled?
+  # We could add other values here if we discover any
+  # but for now all we want to know is if YJIT was enabled at runtime.
+  RubyVM::YJIT.enabled? if defined?(RubyVM::YJIT)
+end
+
+$original_yjit_enabled = yjit_enabled?
```

Comment:
Since it's only written once, how about using a constant instead?
Global variables are kind of soft-deprecated these days.

Reply:
Good point, there are a lot of constants already being used in this way 👍